website/docs/components/form/file-input/partials/specifications/states.md

```diff
@@ -0,0 +1,3 @@
+## States
+
+![Example of file input states](/assets/components/form/file-input/file-input-states.png)
```

Comment:
should we add a state when the file is selected?

Reply:
I could grab separate screenshots showing this for Andrew to add but it's not something that can easily be added to the actual Showcase itself without writing custom JavaScript to try to create a file and set the value somehow since there's no actual `value` attribute on the file input element itself.

What do you think? @didoo @andgen404

Reply:
@KristinLBradley I meant in the screenshot image, not in the showcase.

Reply:
Ok, the screen shots were of the showcase actually so just wanted to be sure.

Reply:
Update: I created screenshots in various browsers which Andrew has now added to the docs.